src/ellip.jl

```diff
@@ -15,14 +15,13 @@ Computes Complete Elliptic Integral of 1st kind ``K(m)`` for parameter ``m`` giv
 \quad \text{for} \quad m \in \left( -\infty, 1 \right] \, .
 ```
 
-External links: [DLMF](https://dlmf.nist.gov/19.2#E8),
-[Wikipedia](https://en.wikipedia.org/wiki/Elliptic_integral#Notational_variants).
+External links: [DLMF](https://dlmf.nist.gov/19.2#E8), [Wikipedia](https://en.wikipedia.org/wiki/Elliptic_integral#Notational_variants).
```

Comment:
You can link to DLMF via X.Y.Z format:

```suggestion
External links: [DLMF 19.2.8](https://dlmf.nist.gov/19.2.8), [Wikipedia](https://en.wikipedia.org/wiki/Elliptic_integral#Notational_variants).
```